src/Form.vue

```diff
@@ -514,11 +525,15 @@ export default {
           // evaluate default()
           if (typeof question.default === "function") {
             try {
-              question._default = await question.default(answers);
-              question.answer = this.getInitialAnswer(question);
-
-              // optimization: avoid repeatedly calling this.getAnswers()
-              answers[question.name] = question.answer;
+		if (question.__origAnswer === undefined) {
+			question._default = await question.default(answers);
+		} else {
+			question._default = question.__origAnswer; 
+		}
+		question.answer = this.getInitialAnswer(question);
+
+		// optimization: avoid repeatedly calling this.getAnswers()
+		answers[question.name] = question.answer;
```

Comment:
did you verify that appltDefaultWhenDirty flag still works?

Reply:
yes it works